pom.xml

```diff
@@ -67,7 +67,6 @@
   <properties>
     <revision>2.5.7</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.138.4</jenkins.version>
```

Comment:
pom.xml changes was a tentative to the build work because 2 days ago the pipeline library has update the jenkins version to 2.222.3 for recomanded version. I would include these changes in the PR if the fix is already in master.

Reply:
anyway thank for attention this PR was there for too long time and every time you release new version I have to rebase update and load to our production a custom build. And also advise all other devops to not update this plugin. Time ago I forgot this and I update plugin. This case the configuration of branchproperty do not work and cause our jenkins production down for a couple of hours to understand what happens and to restore all lost configurations. Very very dangerous